SimPEG/EM/Static/Utils/PlotUtils.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+from scipy.interpolate import LinearNDInterpolator
+import matplotlib.pyplot as plt
+
+def plot2Ddata(xyz, data, vec=None, nx=100, ny=100, ax=None, mask=None, level=None, figname=None, ncontour=10, dataloc=False, contourOpts={}, clim=None):
```

Comment:
That sounds good, we already have some functions that we can plot pseudo section though